docs/writing-rules/experiments/join-mode/overview.md

```diff
@@ -196,7 +196,7 @@ Short for references, `refs` is a list of external rules that make up your code
 
 ### `rule`
 
-This points to an external rule location to use in this join rule. Currently, join mode requires external rules. Additionally, even though Semgrep rule files can typically contain multiple rules under the `rules` key, join mode **will only use the first rule in the file**.
+This points to an external rule location to use in this join rule. Even though Semgrep rule files can typically contain multiple rules under the `rules` key, join mode **will only use the first rule in the provided file**.
```

Comment:
Just checking that I understood this correctly - we no longer require external rules? Does that mean join mode will accept a pattern operator?

Reply:
@armchairlinguist not sure about this edit. @minusworld can you please have a look here?

Reply:
@s-santillan Yes, the document gets into this a little further down, discussing rules vs refs. I've found that both work, so this restriction doesn't apply anymore.